packages/toolkit/connector-kit/src/types.ts

```diff
@@ -93,6 +93,32 @@ export enum MessageType {
 /** @deprecated Use `verificationCodeTypeGuard` instead. */
 export const messageTypesGuard = verificationCodeTypeGuard;
 
+export enum ConnectorConfigFormItemType {
+  Text = 'Text',
+  Number = 'Number',
+  MultilineText = 'MultilineText',
+  Switch = 'Switch',
+  Select = 'Select',
+  Json = 'Json',
+}
+
+const connectorConfigFormItemValueGuard = z.union([z.string(), z.number(), z.boolean()]);
+
+const connectorConfigFormItemGuard = z.object({
+  type: z.nativeEnum(ConnectorConfigFormItemType),
+  key: z.string(),
+  label: z.string(),
+  placeholder: z.string().optional(),
+  required: z.boolean().optional(),
```

Comment:
nit: should `required` have default value?

Reply:
"undefined" is also a "falsy" value.